version.py

```diff
@@ -34,7 +34,7 @@
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = 323  # Last bumped for "GET /streams `is_recently_active`"
+API_FEATURE_LEVEL = 324  # Last bumped for "GET /events `stream_ids`"
```

Comment:
This is not clear enough. Might be better to write something like "... for stream_ids field in stream deletion event".